docs/server/source/http-client-server-api.rst

```diff
@@ -161,6 +165,25 @@ Transactions
    :statuscode 202: The pushed transaction was accepted in the ``BACKLOG``, but the processing has not been completed.
    :statuscode 400: The transaction was malformed and not accepted in the ``BACKLOG``.
 
+.. http:post:: /api/v1/transactions?mode={mode}
+
+   .. note::
+
+       This option is only available when using BigchainDB with Tendermint.
```

Comment:
Since the naming of things isn't settled yet, this should be clear enough for now.